core/src/main/resources/logback.xml

```diff
@@ -24,8 +24,12 @@
     </appender>
 
     <logger name="com.bobocode.bring.web" level="${com.bobocode.bring.web.log.level:-INFO}">
-        <appender-ref ref="CONSOLE" />
-        <appender-ref ref="FILE" />
+        <appender-ref ref="CONSOLE">
+            <turndown>!EXCEPTION: com.bobocode.bring.web.staticcontent.exception.StaticFileNotFoundException</turndown>
+        </appender-ref>
+        <appender-ref ref="FILE">
+            <turndown>!EXCEPTION: com.bobocode.bring.web.staticcontent.exception.StaticFileNotFoundException</turndown>
+        </appender-ref>
```

Comment:
should be removed need to use correct constructor in exception that suppress stack trace